dags/glam.py

```diff
@@ -369,19 +370,7 @@
 )
 
 with dag as dag:
-    extracts_per_channel = SubDagOperator(
-        subdag=extracts_subdag(
-            GLAM_DAG,
-            "extracts",
-            default_args,
-            dag.schedule_interval,
-            table_project_id,
-            billing_project_id,
-            fully_qualified_dataset,
-            dataset_id,
-        ),
-        task_id="extracts",
-    )
+    extracts_per_channel = EmptyOperator(task_id="extracts")
```

Comment:
What's the purpose of the empty operator instead of updating the task dependencies?